docs/02-concepts/01-data-processing/05-policies.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Policies
+hide_table_of_contents: false
+---
+
+# Policies
+Policies define for what purpose a certain [pipeline](../data-pipelines) processes data points.
+
+An important technical
+aspect of a policy is its _retention period_. Every encryption key that is created in the context of a certain pipeline
+will be assigned a policy and an _expiration moment_. What one does with these expirations is up to the organization.
+For certain legal or financial pipelines, it might mean **do not delete encryption key any sooner than**. But for a
+pipeline that deals for instance with a clickstream it might mean **automatically delete after**.
+
+
+## Example
+[uuid]: https://en.wikipedia.org/wiki/Universally_unique_identifier
+
+Policies have a few attributes.
+
+Example:
+```sh
+strm get policy "1 year"
+
+Name: 1 year
+Id: 5c8e653a-8102-4444-ac15-a3d1aa0ff109
+Description:
+Retention(days): 365
+Legal Grounds:
+State: STATE_ACTIVE
+```
+
+**name**: The name of the policy. This has to be unique with your organization. Preferably choose a short word or sentence. This
+name can be used to interact with policies via the cli and the console id
+
+**id**: This [uuid][uuid] is assigned at random upon creation of a policy. It is required when _updating_ policy
+attributes
+
+**description**: A thorough description of the policy. This describes its purpose, and for what kind of pipelines this policy is used.
+
+**retention**: Upon encryption key creation in the context of a certain pipeline, the pipeline policy is applied, and the expiration
+is calculated from the moment the key was first used for encryption plus the number of days retention.
+
+**legal grounds**:  In case the policy is derived from a legal or regulatory text, one could put a reference in this field
+
+**state**: The state of the policy, either `draft`, `active` or `archived`. Policies can only be used in pipelines when they're
+in the `active` state.  Policy attributes can be modified as long as its state is `draft`. Policies can be deleted in
+`draft` or `archived` state.
+
+## Outputs
+Whereas the previous batch key export (for the streams) and the keys csv files for the batch jobs had only two columns
```

Comment:
It's not necessary to talk about how it was IMO. I'd just explain the format in which Batch Jobs expose the data.

Reply:
No because we have a few blog posts that show the old format.